addon/templates/components/polaris-breadcrumbs.hbs

```diff
@@ -1 +1,30 @@
-{{polaris-breadcrumbs/breadcrumb breadcrumb=breadcrumb}}
+{{#if breadcrumb.url}}
+  {{#polaris-unstyled-link
+    key=breadcrumb.content
+    url=breadcrumb.url
+    class="Polaris-Breadcrumbs__Breadcrumb"
+    mouseUp=handleMouseUpByBlurring
+  }}
+    <span data-test-breadcrumb-icon class="Polaris-Breadcrumbs__Icon">
+      {{polaris-icon source="chevron-left"}}
+    </span>
+    <span data-test-breadcrumb-content class="Polaris-Breadcrumbs__Content">
+      {{breadcrumb.content}}
+    </span>
+  {{/polaris-unstyled-link}}
+{{else}}
+  <button
+    key={{breadcrumb.content}}
+    class="Polaris-Breadcrumbs__Breadcrumb"
+    onclick={{action breadcrumb.onAction}}
+    onmouseup={{handleMouseUpByBlurring}}
+    type="button"
+  >
+    <span data-test-breadcrumb-icon class="Polaris-Breadcrumbs__Icon">
+      {{polaris-icon source="chevron-left"}}
+    </span>
+    <span data-test-breadcrumb-content class="Polaris-Breadcrumbs__Content">
+      {{breadcrumb.content}}
+    </span>
+  </button>
+{{/if}}
```

Comment:
How do you feel about using a conditional to avoid duplicating the icon/content stuff between the link and button cases?

Reply:
Discussed this on slack with @tomnez and agreed that it's clearer to leave this as is